activities/TamTamMicro.activity/index.html

```diff
@@ -17,10 +17,12 @@
 <script src="lib/bootstrap-tour-standalone.min.js"></script>
 <script>if (window.module) module = window.module;</script>
 <link href="lib/enyo/enyo.css" rel="stylesheet" type="text/css" />
+<script src="lib/Tone.js"></script>
```

Comment:
Please move this file into the` if (typeof module)` and `if (window.module)` to avoid possible incompatibility with electron as explained [here](https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step9.md#integrate-bootstrap-tour-components)